libadikted/obj_column.c

```diff
@@ -249,6 +249,9 @@ void create_columns_for_slab(struct COLUMN_REC *clm_recs[9],struct LEVOPTIONS *o
     case SLAB_TYPE_GUARDPOST:
       create_columns_slb_guardpost(clm_recs,surr_slb,surr_own,surr_tng);
       break;
+    case 54:
```

Comment:
Add `#define SLAB_TYPE_PURPLE_PATH    0x36` in `obj_slabs.h` and use that instead so magic numbers is avoided.